.ci_support/linux_64_.yaml

```diff
@@ -11,15 +11,15 @@ channel_targets:
 cxx_compiler:
 - gxx
 cxx_compiler_version:
-- '12'
+- '13'
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
 giflib:
 - '5.2'
 jpeg:
 - '9'
```

Comment:
why is this still showing up in the pinnings???

Reply:
I think the presence of the string "jpeg" is just wrongly being parsed out of `meta.yaml` 🤷